index.rst

```diff
@@ -53,6 +53,338 @@
+r[major].[minor].patch``; thus ``r21_0_1`` has the display name ``Release
+r21.0.1``.  Additional components (cycle and extra) are permitted and
+will be appended (with underscores).  ``r21_0_1_c0020.002_20210703``
+becomes ``Release r21.0.1_c0020.002_20210703``.
+
+Semantic Version
+^^^^^^^^^^^^^^^^
+
+The semantic version of a release tag is, actually,
+``[major].[minor].[patch]``.  ``r21_0_1`` has version ``21.0.1``.
+
+Cycle and build version will be added as described above.  Thus:
+``r21_0_1_c0020.002_20210703`` would have the semantic version
+``21.0.1+c0020.002.20210703``.
+
+Notes on "release" category
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+These tags differ from other categories by not having an underscore
+between the type and the release identifier.  There is no reason for
+this other than historic convention.
+
+Weekly
+------
+
+Weekly images are the bread-and-butter workhorse images.  Most users
+will use the latest weekly that has been blessed as "recommended".
+There are three noteworthy things about the weekly images.  First, they
+are the feedstock for "recommended"; second, it is always a particular
+weekly image that is chosen as the basis for a release image; and third,
+we make claim that the weekly image is going to be fit-for-purpose and
+therefore not utterly broken.
+
+Tag Format
+^^^^^^^^^^
+
+Weekly tags are of the form ``w_[year]_[week]``, e.g. ``w_2021_19``.
+They may have additional cycle and rest components;
+``w_2021_19_c0019.001`` is an acceptable weekly tag, for instance, as is
+``w_2021_19_20210513`` or indeed ``w_2021_19_c0019.001_20210513``.
+
+Display Name
+^^^^^^^^^^^^
+
+The display name is ``Weekly [year]_[week]``; ``w_2021_19`` has the
+display name ``Weekly 2021_19``.  As with releases and release
+candidates, additional components are simply appended.  Thus
+``w_2021_19_c0019.001`` would have the display name ``Weekly
+2021_19_c0019.001``.
+
+Semantic Version
+^^^^^^^^^^^^^^^^
+
+A weekly's semantic version is ``[year].[week].0``.  ``w_2021_19`` has
+the version ``2021.19.0``.  Any additional components are used as the
+semver ``build`` string (with underscores replaced by periods), so
+``w_2021_19_c0019.001`` would become ``2021.19.0+c0019.001``.
+
+Daily
+-----
+
+Daily images are, as the name implies, produced every night.  They are
+not guaranteed to work.  They are generally used only by users needing
+bleeding-edge features that haven't made it into a weekly yet.
+
+Tag Format
+^^^^^^^^^^
+
+Daily tags are of the format ``d_[year]_[month]_[day]``; as with weekly
+builds, additional underscore-separated components may exist.
+
+Display Name
+^^^^^^^^^^^^
+
+A Daily display name is ``Daily [year]_[month]_[day]``, so
+``d_2021_05_11`` becomes ``Daily 2021_05_11``.  Additional components
+are handled as for weeklies.
+
+Semantic Version
+^^^^^^^^^^^^^^^^
+
+The version for a daily image is ``[year].[month].[day]``.
+``d_2021_05_11`` is simply ``2021_05_11``.  Additional components go
```

Comment:
```suggestion
``d_2021_05_11`` is simply ``2021.05.11``.  Additional components go
```

Reply:
Cool.  I need to update it for the cycle and rest stuff, which I went ahead and changed.